### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by joegasewicz**
### What
Screenshot 2022-01-27 at 14 39 12

Describe what you have changed and why.
The unit tests will fail until the new public methods from the dp client repo are released

Added the area relations sections (just above the footer) that displays a list of areas within England.
- Visually via the attached image .
- Check the relative links are valid

### How to review
as above ^
Describe the steps required to test the changes.

### Who can review
Team 404
Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: handlers/handlers.go [hunk 1]
```diff
@@ -34,17 +35,50 @@ func GeographyStart(cfg config.Config, rc RenderClient) http.HandlerFunc {
...
+
+		relationsData, relationsErr = c.AreaApi.GetRelations(ctx, accessToken, "", collectionID, areaID, acceptedLang)
+		if relationsErr != nil {
+			log.Error(ctx, "Fetching area relations data", relationsErr)
+			return
+		}
+	}()
+
+	wg.Wait()
+	if err != nil || relationsErr != nil {
```
Comment (joegasewicz) [inline handlers/handlers.go:73]: I have added a helper function for this in another pr
https://github.com/ONSdigital/dp-frontend-area-profiles/blob/7261ac04cd571d4c3cfe8a21930e2738e2108540/utils/requests.go

File: go.mod [hunk 2]
```diff
@@ -3,8 +3,8 @@ module github.com/ONSdigital/dp-frontend-area-profiles
 go 1.16
 
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.7.0
+	//github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
```
Comment (mike-kelly-methods) [inline go.mod:6]: This can be removed.

Comment (joegasewicz) [reply go.mod]: Thanks @mike-kelly-methods will do

**Review by mike-kelly-methods — APPROVED** [review]
👍

**Pull request merged**